### DISCUSSION TITLE
Setting up a Python script for Pyscript

### DISCUSSION THREAD

**Discussion by AdamMonforton** [custom-components/pyscript#323] [Q&A]
I am trying to trigger a Tasker task via Home Assistant. I am following this guide https://www.home-assistant.io/blog/2015/12/10/activating-tasker-tasks-from-home-assistant-using-command-line-switches/ 
which says to create a Python script with this code:
```
import requests
requests.get('[URL]')
```
(The URL gets replaced). I have created my .py file and put this code within the pyscript folder in my Config of Home Assistant. When i go to create my automation and select Call Service. The only pyscript services I see are the Jupyter Kernal and the Reload.I did go back and add @service to the top of my .py file but that hasn't changed anything.

Any advice as to what I am doing wrong?

**Comment by dlashua** [comment]
pyscript and python_script are two very different things.

**Reply by AdamMonforton** [reply]
Yes, which is why I am utilizing pyscript. python_script does not support imports, so I decided to use pyscript for that.

**Reply by dlashua** [reply]
You'll need to spend some more time reading the pyscript documentation. Pyscript is not a drop in replacement for python_script. In addition, most python code you might find will not work in pyscript without some modification. Putting @service at the top of your file will not have the desired effect. You'll need to define a function with a @service decorator, for starters, but you have a bit more to go than that.

The link you provided has complete instructions for doing what you want to do. None of that requires (or even recommends) pyscript.
